nixos/modules/services/web-apps/nextcloud-backup/backup.sh

```diff
@@ -0,0 +1,72 @@
+# Nextcloud backup script, based on https://docs.nextcloud.com/server/latest/admin_manual/maintenance/backup.html.
+set -Eeuo pipefail
```

Comment:
`writeShellApplication` sets `-e` `-u` and `pipefail`.
You can set `-E` by adding `errtrace` to the `bashOptions` for `writeShellApplication`